ToString.js

```diff
@@ -0,0 +1,16 @@
+
+var getColorFromPairNumber = require("./getColorFromPairNumber.js");
+function ToString()
+{   
+    console.log("Pin Number" + " " + " Major" + " " + "Minor");
+    
+    for(var i=1;i<=25;i++)
```

Comment:
Do not hardcode loop limits. Can calculate it from the number of minor and major colors.